versions.txt

```diff
@@ -11,4 +11,5 @@ aws-otel-python-instrumentation=v0.2.0
 aws-otel-dotnet-instrumentation=1.6.0
 
 dcgm-exporter=3.3.3-3.3.1-ubuntu22.04
-neuron-monitor=1.0.0
\ No newline at end of file
+neuron-monitor=1.0.0
+target-allocator=latest
```

Comment:
we should follow the semver versioning here, using latest tag is not recommended